queries/ne.jinja2

```diff
@@ -247,7 +246,7 @@ SELECT
   NULL::jsonb AS __places_properties__,
 
   jsonb_build_object(
-      'min_zoom', GREATEST(scalerank, 5),
+      'min_zoom', scalerank,
       'scalerank', scalerank,
```

Comment:
Confirmed we still need this for YAML logic like: https://github.com/tilezen/vector-datasource/blob/7b7394482ccd72bb8a46f30137203ea49ce974af/yaml/boundaries.yaml#L19. (It's probably possible to refactor it even more, but I think it's fine as-is for now.)